Gemfile

```diff
@@ -7,11 +7,13 @@ gem 'hanami-model', '~> 1.3'
 gem 'pg'
 
 gem 'scrypt'
+gem 'puma'
+
+group :plugins do
+  gem 'hanami-reloader', git: 'https://github.com/DangerDawson/hanami-reloader'
```

Comment:
Out of interest, why are we getting the gym directly from the git repository here rather than the official release on RubyGems?